src/components/MyKiva/BadgeCompleted.vue

```diff
@@ -110,6 +111,9 @@ const utmContent = computed(() => {
 });
 
 const badgeImage = computed(() => {
+	if (!tier.value) {
```

Comment:
When is `tier` undefined?

Reply:
For lending achievements, old badges, they don't have tiers.